README.md

```diff
@@ -249,6 +246,16 @@ git remote add dokku dokku@openafrica.net:ckan
 git push dokku
 ```
 
+### Configure Database
+
+Before you can run CKAN for the first time, you need to run `db init` to initialize your database
+
+```
+dokku enter ckan
+cd src/ckan
+paster db init -c /ckan.ini
+```
```

Comment:
@phillipahereza Config are set up using the `ckanext-envvar` extension.

Doing it as you have will  have `ckan.ini` deleted each time someone makes a change / deploys to dokku.